src/Avalonia.Controls/ToolTip.cs

```diff
@@ -61,6 +61,12 @@ public class ToolTip : ContentControl, IPopupHostProvider
         internal static readonly AttachedProperty<ToolTip?> ToolTipProperty =
             AvaloniaProperty.RegisterAttached<ToolTip, Control, ToolTip?>("ToolTip");
 
+        /// <summary>
+        /// Stores the control that a <see cref="ToolTip"/> is being shown on.
+        /// </summary>
+        internal static readonly AttachedProperty<Control?> ToolTipAdornedControlProperty =
```

Comment:
Can probably be simplified to a standard CLR property since the owner is the `ToolTip` itself and it's internal.

Reply:
Yeah that would make sense.

Reply:
Done.